vscode-stylelint/package.json

```diff
@@ -0,0 +1,210 @@
+          "default": "",
+          "description": "An absolute path to a custom PostCSS-compatible syntax module."
+        },
+        "stylelint.reportNeedlessDisables": {
+          "type": "boolean",
+          "default": false,
+          "description": "Also report errors for `stylelint-disable` comments that are not blocking a lint warning."
+        },
+        "stylelint.stylelintPath": {
+          "type": "string",
+          "default": "",
+          "description": "Supply a custom path to the stylelint module."
+        },
+        "stylelint.packageManager": {
+          "scope": "resource",
+          "type": "string",
+          "enum": [
+            "npm",
+            "yarn",
+            "pnpm"
+          ],
+          "default": "npm",
+          "description": "The package manager you use to install node modules."
+        },
+        "stylelint.validate": {
+          "scope": "resource",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "css",
+            "html",
+            "javascript",
+            "javascriptreact",
+            "less",
+            "markdown",
+            "postcss",
+            "sass",
+            "scss",
+            "source.css.styled",
+            "source.markdown.math",
+            "styled-css",
+            "sugarss",
+            "svelte",
+            "typescript",
+            "typescriptreact",
+            "vue",
+            "vue-html",
+            "vue-postcss",
+            "xml",
+            "xsl"
+          ],
+          "description": "An array of language ids which should be validated by stylelint."
+        }
+      }
+    },
+    "commands": [
+      {
+        "title": "Fix all auto-fixable Problems",
+        "category": "stylelint",
+        "command": "stylelint.executeAutofix"
+      }
+    ],
+    "jsonValidation": [
+      {
+        "fileMatch": "**/.stylelintrc*",
+        "url": "http://json.schemastore.org/stylelintrc.json"
+      }
+    ],
+    "languages": [
+      {
+        "id": "ignore",
+        "filenames": [
+          ".stylelintignore"
+        ]
+      }
+    ],
+    "snippets": [
+      {
+        "language": "css",
+        "path": "./snippets/stylelint-disable.json"
+      },
+      {
+        "language": "less",
+        "path": "./snippets/stylelint-disable.json"
+      },
+      {
+        "language": "postcss",
+        "path": "./snippets/stylelint-disable.json"
+      },
+      {
+        "language": "scss",
+        "path": "./snippets/stylelint-disable.json"
+      }
+    ]
+  },
+  "dependencies": {
+    "fast-diff": "^1.2.0",
+    "kind-of": "^6.0.3",
+    "lodash": "^4.17.15",
+    "path-is-inside": "^1.0.2",
+    "stylelint": "^13.2.1",
+    "vscode-languageclient": "^6.1.1",
+    "vscode-languageserver": "^6.1.1",
+    "vscode-languageserver-textdocument": "^1.0.1",
+    "vscode-languageserver-types": "^3.15.1",
+    "vscode-uri": "^2.1.1"
```

Comment:
Since this is `package.json` of the whole extension, it includes some dependencies that are not needed for the server (like `vscode-languageclient`). They don't hurt apart from taking space and time to install, I guess. Not sure how we could handle that (filter out?)...

Reply:
I don't have the answer on how to solve this.

> They don't hurt apart from taking space and time to install

As you said this is the downside. It is not a deal breaker, but it would be preferable to install only what it needs.

Reply:
I guess in the worst case I can just use sed to remove specific lines.